web-app/src/utilities/AxiosInstance.js

```diff
@@ -2,8 +2,8 @@ import axios from 'axios';
 import moment from 'moment';
 import store from '../store';
 
-const baseURL = process.env.DOMAIN;
-
+//const baseURL = process.env.DOMAIN;
+const baseURL = 'http://localhost:8081';
```

Comment:
You'll need to undo this. In future create a .env file with DOMAIN set up.

Reply:
Just pushed up the fixed AxiosInstance